README.md

```diff
@@ -42,144 +76,321 @@ The only compile time dependency you will need is `okta-sdk-api`.  You will also
+VerifyFactorRequest verifyFactorRequest = client.instantiate(VerifyFactorRequest.class);
+verifyFactorRequest.setPassCode("123456");
+factor.activate(verifyFactorRequest);
 ```
+[//]: # (end: activateFactor)
+
+### Verify a Factor
+
+[//]: # (method: verifyFactor)
+```java
+Factor factor = user.getFactor("factorId");
+VerifyFactorRequest verifyFactorRequest = client.instantiate(VerifyFactorRequest.class);
+verifyFactorRequest.setPassCode("123456");
+VerifyFactorResponse response = factor.verify(verifyFactorRequest);
+```
+[//]: # (end: verifyFactor)
+
+### List all Applications
 
-## Calling other Okta endpoints
+[//]: # (method: listApplication)
+```java
+ApplicationList applications = client.listApplications();
+```
+[//]: # (end: listApplication)
 
-Is the endpoint you want to call not supported in this SDK yet? Use `client.getDataStore().http()` to make http calls with a convenient builder interface.  The following example sets a user's password using the [set password](https://developer.okta.com/docs/api/resources/users#set-password) end point (which isn't supported yet).
+### Get an Application
 
+[//]: # (method: getApplication)
 ```java
-String userId = "some-user-id";
+Application app = client.getApplication("appId");
+```
+[//]: # (end: getApplication)
 
-// Create a resource object
-ExtensibleResource userPasswordRequest = client.instantiate(ExtensibleResource.class);
-userPasswordRequest.put("credentials", client.instantiate(UserCredentials.class)
-                        .setPassword(client.instantiate(PasswordCredential.class)
-                            .setValue("aPassword1!".toCharArray())));
+### Create a SWA Application
 
-User result = client.getDataStore()
-                .http()
-                    .setBody(userPasswordRequest)
-                    .addQueryParameter("key", "value")
-                    .post("/api/v1/users/"+ userId, User.class);
+[//]: # (method: createSwaApplication)
+```java
+SwaApplication swaApp = client.instantiate(SwaApplication.class)
+    .setSettings(client.instantiate(SwaApplicationSettings.class)
+    .setApp(client.instantiate(SwaApplicationSettingsApplication.class)
+      .setButtonField("btn-login")
+      .setPasswordField("txtbox-password")
+      .setUsernameField("txtbox-username")
+      .setUrl("https://example.com/login.html")));
 ```
+[//]: # (end: createSwaApplication)
+
+### Call other API endpoints
 
-## Proxy Configuration
+Not every API endpoint is represented by a method in this library. You can call any Okta management API endpoint using this generic syntax:
 
-Use the standard [Java proxy system properties](https://docs.oracle.com/javase/8/docs/technotes/guides/net/proxies.html) ([Spring documentation](https://docs.spring.io/spring-integration/reference/html/http.html#http-proxy)) to configure proxy support.
+[//]: # (method: callAnotherEndpoint)
+```java
+ExtensibleResource protocol = client.instantiate(ExtensibleResource.class);
```

Comment:
Is there another example you can use? I think people might think this is using `OAuth://` instead of HTTP 😛

Reply:
IdP creation is a popular example, but I can reword it a little to make it less confusing (good point about the protocol)